pkgs/tools/security/gen-oath-safe/default.nix

```diff
@@ -0,0 +1,44 @@
+{ coreutils, makeWrapper, openssl, libcaca, qrencode, fetchFromGitHub, yubikey-manager, python, stdenv, ... }:
+
+stdenv.mkDerivation {
+  name = "gen-oath-safe-2017-01-23";
+  src = fetchFromGitHub {
+    owner = "mcepl";
+    repo = "gen-oath-safe";
+    rev = "fb53841";
+    sha256 = "0018kqmhg0861r5xkbis2a1rx49gyn0dxcyj05wap5ms7zz69m0m";
+  };
+
+  phases = [
```

Comment:
Not sure what the difference should be as `buildPhase = ":";` may also override behavior in the future which could break the build. Newly introduced phases may also break the build.
if explicitly setting phases is deemed as a bad practice then it might be a good idea to add the recommendation somewhere in the nixpkgs manual.

Nevertheless i will change the package build according to your recommendations.